api/spec/controllers/mno_enterprise/jpi/v1/admin/account_transactions_controller_spec.rb

```diff
@@ -34,11 +34,15 @@ module MnoEnterprise
 
       describe 'creation' do
         context 'success' do
+          let(:data) { JSON.parse(response.body) }
+
           before { stub_api_v2(:get, '/tenant', tenant) }
-          before { subject }
```

Comment:
you can't put a before { subject } before the shared example groups, or else things get run in an incorrect order.